tbmalt/io/dataset.py

```diff
@@ -1,66 +1,109 @@
 # -*- coding: utf-8 -*-
-from typing import Dict, List
+from typing import Dict, List, Union
 import random
 
-import h5py
+from h5py import Group
 import torch
 from torch.utils.data import DataLoader as Loader
 from torch import Tensor
 
-from tbmalt.common.batch import deflate, pack
+from tbmalt.common.batch import pack
 
 
 class Dataloader(Loader):
-    """"""
+    """A class to read and load data.
 
-    def __init__(self, atomic_numbers: Tensor, positions: Tensor,
```

Comment:
Some information about the structure of the arguments might be useful. I am unsure of what exactly is stored within `dataset`.